### DISCUSSION TITLE
Seeding limit configuration for auto removal

### DISCUSSION THREAD

**Discussion by adyanth** [jesec/flood#347] [Q&A]
Type: Documentation Issue

## Summary

Does flood support configuring seeding limits? 
The client is transmission, and I am using this with sonarr which should support removal of downloads after completion, but it seems to wait till the seeding limit is hit. 
Is there a way to change transmission's seed limit from flood?

**Comment by jesec** [comment]
Moving to discussions.

**Comment by jesec** [comment]
You should connect Sonarr with Flood directly. 

Sonarr will then provide you the options to set appropriate criteria.

**Reply by adyanth** [reply]
Hey @jesec 
I have sonarr talking to Flood itself. Currently, it just tags it at sonarr-complete. Maybe that is an issue with sonarr, but generally, since this is front-ending transmission, it would a nice to have feature.

While we are here, could you tell the path to the config file which I can bind mount in docker so that the configs are not lost on a recreate?

**Reply by jesec** [reply]
![image](https://user-images.githubusercontent.com/11585141/123950765-9cb0e700-d9d6-11eb-845f-b4df308fbc7e.png)

It is the path to runtime directory, by default it is `/home/download/.local/share/flood`.
